### ISSUE TITLE
Regression in 7.3.0 when choosing SASS as preprocessor

### ISSUE THREAD

**Issue by wkurniawan07** [angular/angular-cli#13550]
# 🐞 Bug report

### Command (mark with an `x`)

```
- [x] new
- [x] build
- [x] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Is this a regression?

Yes, the previous version in which this bug was not present was: All 7.x before 7.3.0

### Description

When choosing Sass as the stylesheet preprocessor, CLI produces `styles.scss` but `angular.json` expects `styles.sass`.
Previous available choice of "Scss" is gone.

## 🔬 Minimal Reproduction

1. `ng new`
1. Choose `Sass` as the stylesheet format
1. `ng build` or `ng serve`

## 🔥 Exception or Error
```
ERROR in multi ./src/styles.sass
Module not found: Error: Can't resolve '[...]/src/styles.sass' in '[...]'
```
`[...]` above refers to the project folder.

**Comment by webcourse-ng** [comment]
Thank you, wkurniawan07! I spent a good half a day wrestling this one... checked my setup environment in a million places after not finding anything on Google yesterday. Even started a thread on Stack Overflow: https://stackoverflow.com/questions/54463979/angular-7-cli-generated-project-throws-error-in-multi-src-styles-sass

**Comment by borriej** [comment]
I have the same bug. Just updated my CLI to latest ```7.3.0``` and then created a new project... choose sass.. (which is scss)

but there is a typo in the cli source code (i guess)

```
  "projects": {
    "angular-skeleton": {
      "schematics": {
        "@schematics/angular:component": {
          "style": "sass"
        }
      },
      "architect": {
        "build": {
          "builder": "@angular-devkit/build-angular:browser",
          "options": {
            "styles": [
              "src/styles.sass"
            ],
```

```sass``` should be ```scss``` here

**Comment by DanJFletcher** [comment]
I have zero experience with the internals of Angular but could this bug be caused by line 129 in this file? https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/ng-new/schema.json

I'm wondering if `value` is supposed to be `scss` and not `sass`. I have no idea how the templates actually get generated so I could be way off. But without running the tool in a local dev environment it's my best guess.

**Comment by karamjeetiron** [comment]
I have number of files with extension .scss. Do i have to change the extension to scss to sass?

**Comment by alan-agius4** [comment]
@karamjeetiron, no you don't have too. 

The extension should be `scss` and not `sass`. We stopped offering `sass` as  file extension for new projects.

See https://github.com/angular/angular-cli/pull/13444

**Comment by clydin** [comment]
This has been corrected in CLI 7.3.1 and all newly generated applications will contain the correct file extensions.  Existing projects suffering from this issue will need to manually update the file extension in `angular.json` as noted above.
